lms/templates/wiki/base.html

```diff
@@ -80,7 +80,7 @@
               <div id="alert_stat_bar"></div>
               <script type="text/javascript">
                 new StatusAlertRenderer(
-                  "{{ message }}",
+                  "{{ message|safe }}",
```

Comment:
Thank you for the fix.

1. I _think_ this is correct, although it is misleading. Is there a way to add a test to ensure that a dangerous message (e.g. including `<script>` or something) is properly displayed?  
2. Related, I don't know syntax for commenting in these files, but maybe we should add a comment like the following:
```
The message is not actually safe, but StatusAlertRenderer uses react which adds escaping,
so marking as safe keeps the message from being double-escaped.
```

The reason for the comment and the test is that if that it would be simple to refactor this code in a way where it no longer would be safe if it weren't passed to something that adds the safety later.

Let me know your thoughts.

Reply:
@robrap done